internal/config/config.go

```diff
@@ -228,6 +228,10 @@ const (
 
 	GoogleApplicationCredentials = "GOOGLE_APPLICATION_CREDENTIALS"
 
+	AliyunAccessKeyID     = "OSS_ACCESS_KEY_ID"
```

Comment:
using Aliyun and Alicloud create inconsistency in naming, I suggest it should use alicloud everywhere